core/storage/exploration/gae_models.py

```diff
@@ -239,59 +239,6 @@ def has_reference_to_user_id(cls, unused_user_id):
         return False
 
 
-class ExplorationMathRichTextInfoModel(base_models.BaseModel):
```

Comment:
Have we deleted this models once the svgs are generated for an exploration? If not, then don't remove this model now, write a simple one-off job to delete these models. [In a separate PR we will remove these model related code!]

Reply:
We have a one-off job in develop `ExplorationMathRichTextInfoModelDeletionOneOffJob` which should be run in prod server and test-server in this release after the math svgs are generated in the prod server.
cc: @aks681 .